clientgui/ViewTransfers.cpp

```diff
@@ -47,8 +47,9 @@
 #define COLUMN_PROGRESS             2
 #define COLUMN_SIZE                 3
 #define COLUMN_TIME                 4
-#define COLUMN_SPEED                5
-#define COLUMN_STATUS               6
+#define COLUMN_TOCOMPLETION         5
```

Comment:
Ok, I will fix.  Should I adjust where COLUMN_TOCOMPLETION is located in the rest of the code?  For example in switch cases, move it to the end?  I know it will function the same, just keeping formatting consistent.

Reply:
No, this is not needed. Just IDs should be fixed

Reply:
Fixed.